File: API/ODDO.Data/Models/OrderModel.cs
using ODDO.Data.Enums;

namespace ODDO.Data.Models; 

public class OrderModel: BaseModel {
    public Status Status { get; set; }
    public WaiterModel Waiter{ get; set; }
    public TableModel Table { get; set; }
    
    public HashSet<OrderProductModel> Products { get; set; }
}

File: APP/app/src/main/java/art/mohregregs/oddo/network/models/OrderModel.kt
package art.mohregregs.oddo.network.models

import art.mohregregs.oddo.network.enums.Status

class OrderModel(
    var id: Int = -1,
    var status: Int = 0,
    var waiter: WaiterModel = WaiterModel(),
    var table: TableModel?,
    var products: List<OrderProductModel>
) {
}